djangoproject/templates/aggregator/local-django-community.html

```diff
@@ -21,7 +21,8 @@ <h2>Local Django Communities</h2>
       <h2>{{ local_django_community.grouper.title }} <a class="plink" href="#{{ local_django_community.grouper.title | lower }}-meetups">¶</a></h2>
       <ul>
         {% for django_community in local_django_community.list %}
-          <li>
+          <ul>
+            <img src={{ django_community.og_image }} />
```

Comment:
Quotes for src parameter are missing, If `og_image='https://some/url.jpg onload=alert(document)'`we have working XSS attack! Never trust anything that comes from users, someone could prepare website with some malicious code in og:image field.